src/neo/Trie/MPT/MPTNode/HashNode.cs

```diff
@@ -0,0 +1,52 @@
+using Neo.IO;
+using Neo.IO.Json;
+using System;
+using System.IO;
+
+namespace Neo.Trie.MPT
+{
+    public class HashNode : MPTNode
+    {
+        public byte[] Hash;
```

Comment:
Is not a fixed size `Hash`? I think it's better to use `UInt256` because it's already optimized

Reply:
ok